calico_containers/tests/st/test_add_ip.py

```diff
@@ -0,0 +1,69 @@
+from sh import ErrorReturnCode
+from functools import partial
+
+from test_base import TestBase
+from docker_host import DockerHost
+
+
+class TestAddIp(TestBase):
+    def test_add_ip(self):
+        host = DockerHost('host')
+
+        calicoctl = "/code/dist/calicoctl %s"
+        host.execute(calicoctl % "node --ip=172.17.8.10")
+        host.execute(calicoctl % "profile add TEST_GROUP")
+
+        self.assert_powerstrip_up(host)
+
+        host.execute("docker run -e CALICO_IP=192.168.1.1 -tid --name=node1 busybox", docker_host=True)
```

Comment:
Thinking about this more, I find the name `docker_host` to be unhelpful.  What this option actually does is determine whether to route the docker command via Powerstrip or not.  Maybe `use_powerstrip` is a better name.